model/Image.php

```diff
@@ -233,10 +247,10 @@ public function SetRatioSize($width, $height) {
 		$heightRatio = $height / $this->height;
 		if( $widthRatio < $heightRatio ) {
 			// Target is higher aspect ratio than image, so check width
-			if($this->isWidth($width) && !Config::inst()->get('Image', 'force_resample')) return $this;
+			if($this->isWidth($width)) return $this;
```

Comment:
However it will now call the `ResizedImage` function in the case where some dimension ins the same on the image, rather than the original format name. The file name generated will not be the same as if it was called through the original function.

I'm pretty sure thats a backwards breaking API change and wouldn't be appropriate for 3.2 in that case?